packages/suite/src/utils/wallet/formatCurrencyAmount.ts

```diff
@@ -0,0 +1,15 @@
+export const formatCurrencyAmount = (amount: number, locale = 'en'): string => {
+    if (
+        typeof amount !== 'number' ||
+        Number.isNaN(amount) ||
+        !Number.isFinite(amount) ||
+        amount > Number.MAX_SAFE_INTEGER ||
+        amount < Number.MIN_SAFE_INTEGER
+    ) {
+        return '';
+    }
+
+    return Intl.NumberFormat(locale, { style: 'decimal', maximumFractionDigits: 20 }).format(
```

Comment:
i think we should avoid using globals in utils (or anywhere else if possible)
`Intl` should be imported

Reply:
Likely it's not needed as it's a [native JS API](https://github.com/trezor/trezor-suite/pull/5526#discussion_r933244711)